### DISCUSSION TITLE
What does "armorvalue" do?

### DISCUSSION THREAD

**Discussion by BiffW** [FakeFishGames/Barotrauma#11875] [Code questions and modding support]
This property is defined on `Wearable` element of ballastic helmet.
I've searched references and couldn't find anything about this property.

**Comment by pantherpusher** [comment]
literally nothing I kid you not, it's vestigial

**Reply by BiffW** [reply]
Thanks for replying. I thought it was something like a priority value to tell AI which helmet to put on in certain cases.
